### DISCUSSION TITLE
Private packages visibility on public profile

### DISCUSSION THREAD

**Discussion by ericlery** [community/community#14574] [Packages] [answered]
I have 2 private packages on my profile, and the number is visible in my public profile, though they are rightly not visible in detail. 

I have several private repositories and the number is not visible, and that's the behavior I expect. The number of private packages should not be visible as it is for private repositories.

It's not a coherent behavior.

Of course I checked in a private window browser without being logged.

![Screenshot from 2022-04-08 14-29-31](https://user-images.githubusercontent.com/7095073/162492272-e6119a80-22e9-4573-a9ba-0a55b8ae2cb9.png)

**Comment by FlorianWendelborn** [comment]
This appears to be fixed now:
![image](https://user-images.githubusercontent.com/1133858/173622799-86dfd9dd-419b-43fc-9809-e9cb39259f2a.png)

@ericlery if it’s fixed for you too, then feel free to mark this as solved

**Comment by ericlery** [comment, marked as answer]
It had indeed been solved.

**Comment by jafludo** [comment]
Very nice !
